packages/js/migrations.json

```diff
@@ -96,6 +96,15 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "17.0.0": {
+      "version": "17.0.0-beta.0",
```

Comment:
Can you change this to `17.0.0-rc.2` please?

Reply:
sure, on it!